Automattic/vip-cli
# SQL Import: Print the status every .1 seconds

## Description

Currently, we only update the "spinner" glyph whenever we get a result from the server. Since our interval is ~ 5 seconds, that means the spinner is not very dynamic.

This change decouples the status printing from the fetch results and prints on a 1 second interval.

The interval is cleared when the promise resolves (or is rejected).

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-import-sql.js`
    * The spinner should update every second (when a task is "running")
    * The command should exit correctly on success & error
1. Run `./dist/bin/vip-import-sql-status.js` while there is an ongoing import
    * The spinner should update every second (when a task is "running")
    * The command should exit correctly on success & error
